.osparc/jupyter-neuron/metadata.yml

```diff
@@ -3,16 +3,24 @@ thumbnail: https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Jupyter_log
 description:
   "[JupyterLab](https://jupyter.org/) with **math** kernels in [octave](https://www.gnu.org/software/octave/index)
   (largely compatible with Matlab code) and python. 
-  Python kernel includes includes neuron simulation packages like [NEURON](https://neuron.yale.edu/neuron/), 
-  [NetPyNE](http://netpyne.org/), [Brian2](https://brian2.readthedocs.io/en/2.0b4/index.html) and 
-  [brian2tools](https://pypi.org/project/brian2tools/).
+  Python kernel includes:
+  - [NEURON](https://neuron.yale.edu/neuron/)
+  - [NEST](https://nest-simulator.readthedocs.io/en/stable/ref_material/pynest_apis.html)
+  - [TVB](https://pypi.org/project/tvb/)
+  - [PyNN](https://neuralensemble.org/PyNN/)
+  - [BluePyOpt](https://github.com/BlueBrain/BluePyOpt#readme)
+  - [Brian2](https://brian2.readthedocs.io/en/stable/) 
+  - [brian2tools](https://pypi.org/project/brian2tools/)
+  - [NetPyNE](http://netpyne.org/)
+  - [pyNeuroML](https://docs.neuroml.org/Userdocs/Software/pyNeuroML.html)
+
   It also includes math packages like [numpy](https://numpy.org/), [pandas](https://pandas.pydata.org/),
   [matplotlib](https://matplotlib.org/), [plotly](https://plotly.com/python/) and
-  [scipy](https://www.scipy.org/) packages (see [requirements.in](https://github.com/ITISFoundation/jupyter-neuron/kernels/python-neuron/requirements.in)).
+  [scipy](https://www.scipy.org/) packages (see [requirements.in](https://github.com/ITISFoundation/jupyter-neuron/blob/main/kernels/python-neuron/requirements.in)).
   "
 key: simcore/services/dynamic/jupyter-neuron
-version: 2.0.0
-integration-version: 2.0.0
+version: 2.0.1
+integration-version: 2.0.1
```

Comment:
Could you please put back `integration-version: 2.0.0`? We have a little problem with the Makefile: it updated also this one, but it shouldn't (since it found 2.0.0, which by coincidence is the same as the `version` of the service.